metadata-integration/java/spark-lineage/spark-smoke-test/smoke.sh

```diff
@@ -40,9 +40,9 @@ echo "--------------------------------------------------------------------"
 
 pwd ../../../
 
-DATAHUB_TELEMETRY_ENABLED=false  \
-DOCKER_COMPOSE_BASE="file://$(cd "$(dirname "${DIR}/../../../../../")"; pwd)" \
-datahub docker quickstart --build-locally --dump-logs-on-failure
+#DATAHUB_TELEMETRY_ENABLED=false  \
```

Comment:
Are these properties getting passed in elsewhere? Especially telemetry, we don't want our smoke tests to impact the numbers.

Reply:
Yes, check https://github.com/datahub-project/datahub/blob/24be154e9c3ec2fc1b4c18aae50b65df13207ccd/docker/build.gradle#L23 already in master.

Reply:
We can remove this commented code too